packages/react/src/components/FocusTrapZone/FocusTrapZone.test.tsx

```diff
@@ -425,28 +445,41 @@ describe('FocusTrapZone', () => {
         ReactTestUtils.Simulate.focus(firstBumper);
       });
       expect(lastFocusedElement).toBe(buttonF);
+
+      removeTestContainer();
     });
   });
 
   describe('Tab and shift-tab do nothing (keep focus where it is) when the FTZ contains 0 tabbable items', () => {
+    let cleanupTest: Function;
+
+    afterEach(() => {
+      cleanupTest();
+    });
+
     function setupTest(props: IFocusTrapZoneProps) {
-      const topLevelDiv = (ReactTestUtils.renderIntoDocument(
-        <div onFocusCapture={_onFocus}>
-          <button className={'z1'}>z1</button>
-          <FocusTrapZone className={ftzClassname} forceFocusInsideTrap={true} {...props}>
-            <button className={'a'} tabIndex={-1}>
-              a
-            </button>
-            <button className={'b'} tabIndex={-1}>
-              b
-            </button>
-            <button className={'c'} tabIndex={-1}>
-              c
-            </button>
-          </FocusTrapZone>
-          <button className={'z2'}>z2</button>
-        </div>,
-      ) as unknown) as HTMLElement;
+      const { removeTestContainer, testContainer: topLevelDiv } = createTestContainer();
+      cleanupTest = removeTestContainer;
```

Comment:
**suggestion:** instead of cross referencing outer scope variables and cleaning things within `afterEach` can we rather make it scoped on spec level? meaning -> returning `removeTestContianer` from this `setup` and calling that per test as needed.